redis_enterprise/README.md

```diff
@@ -0,0 +1,65 @@
+# Agent Check: Redis Enterprise
+
+## Overview
+
+[Redis Enterprise][1] Software is a self-managed data platform that unlocks the full potential of Redis at enterprise scale. The speed you know and love, plus compliance, reliability, and unmatched resiliency for enterprise needs.
+
+## Setup
+
+### Installation
+
+To install the Redis Enterprise check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/python/)
+ on any machine.
+
+2. Checkout the [repository][2] 
+
+3. Run `ddev release build redis_enterprise` to build the package.
+
+3. [Download the Datadog Agent][3].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/redis_enterprise/dist/<ARTIFACT_NAME>.whl`. For information regarding starting and stopping the agent, see [start, stop, and restart the agent][5]. For other information about the agent, see [agent status and information][6].
```

Comment:
The steps should look similar to the steps in this [article](https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=agentv721v621#setup). 

Run the following command to install the Agent integration:

datadog-agent integration install -t datadog-<INTEGRATION_NAME>==<INTEGRATION_VERSION>
The version for the integration can be found in the respective changelog on the integration’s Github repository

Configure your integration similar to core [integrations](https://docs.datadoghq.com/getting_started/integrations/).

[Restart the Agent](https://docs.datadoghq.com/agent/configuration/agent-commands/#restart-the-agent).

Reply:
I pointed them at start, stop and restart the agent but can change it to simply restart if you prefer.